browser-switch/src/main/java/com/braintreepayments/api/BrowserSwitchClient.java

```diff
@@ -67,7 +68,11 @@ public void start(@NonNull FragmentActivity activity, @NonNull BrowserSwitchOpti
             customTabsInternalClient.launchUrl(activity, browserSwitchUrl, launchAsNewTask);
         } else {
             Intent launchUrlInBrowser = new Intent(Intent.ACTION_VIEW, browserSwitchUrl);
-            activity.startActivity(launchUrlInBrowser);
+            try {
+                activity.startActivity(launchUrlInBrowser);
+            } catch (ActivityNotFoundException e) {
```

Comment:
Opted for this approach rather than a check if able to perform intent to avoid any false negatives on devices having browsers.